3.9/graphs-smart-graphs.md

```diff
@@ -83,6 +94,13 @@ This ensures that graph traversals, shortest path, and k-shortest-paths queries
 can be executed locally on a DB-Server, achieving improved performance for
 these type of queries.
 
+Benefits of Hybrid Disjoint SmartGraphs
+-------------------------------
+
+Hybrid Disjoint SmartGraphs benefit of all advantages which are present in
+Hybrid SmartGraphs. The rules of Disjoint SmartGraphs, described in the chapter
+above, stay true.
+
```

Comment:
If we now add a comment how edge collections in a hybrid smart graph are replicated, we should also note here that in this case they are replicated like the disjoint smart collections (if I got that correctly), rather than like satellite collections.